pkg/consts/pod.go

```diff
@@ -46,3 +46,8 @@ const (
 	// PodAnnotationResourcePoolKey is a const variable for pod annotation about resource pool name
 	PodAnnotationResourcePoolKey = "katalyst.kubewharf.io/resource_pool"
 )
+
+// PodAnnotationResourcePackageKey is a const variable for pod annotation about resource package name
+const (
+	PodAnnotationResourcePackageKey = "katalyst.kubewharf.io/resource_package"
```

Comment:
cc @luomingmeng @XudongLiuHarold 

Is it more appropriate to use label? Considering that we can get a class of pods through label selector.

Reply:
@luomingmeng I am ok with that, wdyt?